src/main/java/com/example/SOPSbackend/dto/EditPatientAccountDto.java

```diff
@@ -12,16 +12,7 @@
 @Setter
 @NoArgsConstructor
 @AllArgsConstructor
-public class EditPatientAccountDto {
```

Comment:
Co to wsm robi?

Reply:
Wspólne pola dwóch klas wyciągnąłem do superklasy, żeby nie duplikować kodu. O to chodzi?

Reply:
Klasa w tym pliku służy do zmiany danych konta pacjenta (czyli razem z hasłem)

Reply:
mogę zamknąć ten wątek?

Reply:
Si